app/views/sub_category/index.html.erb

```diff
@@ -13,21 +13,23 @@
           </div>
         </div>
       </div>
-    <div class="background-wide background-cream">
-      <div class="govuk-grid-row govuk-!-padding-top-9 govuk-!-padding-bottom-7">
-        <div class="govuk-grid-column-full">
-          <%= render 'layouts/components/small', text: @sub_category[:collection_name] %>
-          <%= render 'layouts/components/headings/one', text: @sub_category[:title] %>
-          <% @sub_category[:description].each do |description| %>
-            <%= render 'layouts/components/paragraph', text: description[:data][:text] %>
-          <% end %>
+      <div class="background-wide background-cream">
+        <div class="govuk-grid-row govuk-!-padding-top-9 govuk-!-padding-bottom-7">
+          <div class="govuk-grid-column-full">
+            <%= render 'layouts/components/small', text: @sub_category[:collection_name] %>
+            <%= render 'layouts/components/headings/one', text: @sub_category[:title] %>
```

Comment:
Could use the parent heading renderer here for consistency

Reply:
For this we can't use the heading renderer because in Contentful the `title` field is currently set to a generic text field, it isn't a heading content type. We could potentially change it? 🤔